controls/radchartview/getting-started/introduction.md

```diff
@@ -167,10 +167,20 @@ In order to use the chartview components in code, you will need to add the requi
 
 Although you can easily populate the __RadCartesianChart Series__ directly with DataPoint objects, in most real-life applications, you will populate the series with business data. This is why all series types expose an __ItemsSource__ property. You can bind that property to a collection of business items. And if you use a collection that implements the __INotifyCollectionChanged__ interface, then any changes within the business data will be immediately reflected in the appropriate __RadChartView__ components and vice versa. You can find more information in the [Populating with Data]({%slug radchartview-series-databinding%}) section of our documentation.
 
+## Caching mechanism
+
+When the chart is loaded it saves the size of most of its elements, as labels and data point visuals to optimize the performance and it uses this cache size. In order to update these elements, you need to update the chart's API. For example, if you change the axis range, add a new data point, remove/add series, etc. __Example 6__ demonstrates one of the approaches which can be used to update the chart's API.

```

Comment:
Use the only word 'save' or the word 'cache'. Currently, it could be a bit confusing "..chart is loaded it saves the size.." and then ".. and it uses this cache size". You can consider something like this: "..chart is loaded it caches the size.." and "..and it uses the cached size".

Also, "you need to update the chart's API" is not accurate. We won't need to update the API and include new code in it. Instead you can tell that "you need to update a property or call a method from the chart's API."

Also, give an example when the caching mechanism could lead to issues. For example, in a scenario with custom label template where the size of the element in the DataTemplate changes at run-time. Because the size of the ContentPresenter that wraps the elements in the DataTemplate is cached, the element will be clipped if its size becomes bigger than the size of the ContentPresenter.